src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationCtorCall.java

```diff
@@ -115,7 +115,7 @@ public Valid(InputIndentationCtorCall obj, float arg) { //indent:4 exp:4
     public Valid(InputIndentationCtorCall obj, double arg) { //indent:4 exp:4
       obj. //indent:6 exp:6
           super( //indent:10 exp:10
-            x -> arg); //indent:12 exp:12
+          x -> arg); //indent:10 exp:12,14 warn
```

Comment:
please keep existing cases as is, and create new one is required

Reply:
Done